site/plugins/kirby-vite/routes.php

```diff
@@ -45,9 +48,12 @@
                 $pageId = site()->homePageId();
             }
 
-            $page = page($pageId) ?? site()->errorPage();
-            $html = Page::render($page, 'html');
+            $page = kirby()->page($pageId);
+            if (!$page || !$page->isReadable()) {
+                $page = site()->errorPage();
```

Comment:
Same as comment above.

Reply:
Just to clarify, you mean line 25 and 53, correct?